content/oci/oci.go

```diff
@@ -454,6 +455,76 @@ func (s *Store) writeIndexFile() error {
 	return os.WriteFile(s.indexPath, indexJSON, 0666)
 }
 
+// reloadIndex reloads the index and updates metadata by creating a new store.
+func (s *Store) reloadIndex(ctx context.Context) error {
+	newStore, err := NewWithContext(ctx, s.root)
+	if err != nil {
+		return err
+	}
+	s.index = newStore.index
+	s.storage = newStore.storage
+	s.tagResolver = newStore.tagResolver
+	s.graph = newStore.graph
+	return nil
+}
+
+// GC removes garbage from Store. The garbage to be cleaned are:
```

Comment:
Need to mention that unsaved index will be lost.

Reply:
Added documentation to mention this.